helm_deploy/send-legal-mail-to-prisons/Chart.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-service
     alias: gotenberg
-    version: 1.6.0
+    version: 2.6.4
```

Comment:
updated to version 2.6.4 to update kubernetes version to 1.25. Can you think of any issues? Any other changes needed?